mindmeld/components/request.py

```diff
@@ -228,9 +228,10 @@ def dm_params(self, handler_map):
             return {"target_dialogue_state": None}
         return {"target_dialogue_state": target_dialogue_state}
 
-    def nlp_params(self):
+    def validate_nlp_params(self):
         """
-        Validate time zone, timestamp, and dynamic resource parameters.
+        Validate time zone, timestamp, and dynamic resource parameters and return the params as a
```

Comment:
... plus language and locale. Seems there should be a better conceptual description for the parameters grouped together here. And we should keep the method above with a parallel name (=> validate_dm_params)

Reply:
@vijay120 : what are the reasons for grouping these together?